src/api/flow/flow.js

```diff
@@ -0,0 +1,87 @@
+import flow from 'lodash/flow';
+import curry from 'lodash/curry';
+import Immutable from 'immutable';
+
+import { throwInDev } from '../throwInDev';
+import { Node } from './..';
+
+/**
+ * check if node exist in flow
+ * @param {FlowState} state
+ * @param {string} nodeId
+ * @return {bool} true if node exist
+ */
+export const isNodeExist = curry((state, nodeId) => state.hasIn(['nodes', nodeId]));
```

Comment:
imho, nodeExists or hasNode would be better naming

Reply:
i like hasNode, since it will be used in this kind of way
`Flow.hasNode(state, nodeId)`